src/components/Sponsors/Sponsors.css

```diff
@@ -0,0 +1,64 @@
+.sponsors-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: 50px 0px;
+}
+
+.sponsors-title {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 20px;
+}
+
+.box {
+  display: flex;
+  justify-content: center;
```

Comment:
`justify-content` actually only applies to child elements - because this element doesn't have any child elements, we can remove this.